src/app/reporting/Engine.h

```diff
@@ -119,6 +127,8 @@ class Engine
      *
      */
     uint32_t mCurReadHandlerIdx = 0;
+
+    ClusterInfo * mpChangePathList = nullptr;
```

Comment:
Need comments on what this variable is tracking, and why it exists. This is a very key, important list, and the distinction between this and the ClusterInfo list tracked in each ReadHandler has to be called out.

Reply:
Just added, thanks